### PR TITLE
:zap: filter table data based on user input

### LINKED ISSUE #1008: Update asset id search to filter based off user input
ℹ️ # Overview
On `ManageToken.tsx` `CreateTokenSale.tsx` and `ManageTokenSale.tsx` When a user inputs a number in the "ASA Asset Id" field, the search table should filter by number.

### PR DESCRIPTION
**Pull request by treasurechic**
# ℹ Overview
closes #1008 

### 🔐 Acceptance:

- [ ] `yarn test` passes

### CODE REVIEW

File: hooks/useManageToken.tsx [hunk 1]
```diff
@@ -129,7 +129,7 @@ const useManageToken = () => {
   const rowData = useMemo(() => {
     if (activeWallet) {
       return activeWallet['created-assets']
-        .filter((as) => !as.deleted)
+        .filter((as) => !as.deleted && as.index.toString().includes(formData.assetId))
```
Comment (ericsharma) [inline hooks/useManageToken.tsx]: This is a good start but the filtering method should  filter from the beginning of `formData.assetId` so `includes()` is not the right method. When a user types an assetId into the form they are expecting the filter to list assetId's which first character matches their input.

**Review by ericsharma — CHANGES_REQUESTED** [review]
The PR will be good to go once the changes related to the `includes()` method is implemented in the three corresponding components

File: hooks/useManageToken.tsx (hunk 1 above)
Comment (treasurechic) [reply hooks/useManageToken.tsx:132]: "When a user types an assetId into the form they are expecting the filter to list assetId's which first character matches their input." -- this is exactly what includes does. Did you test?

Comment (treasurechic) [reply hooks/useManageToken.tsx]: Ok, will update that

**Review by ericsharma — APPROVED** [review]

**Pull request merged**